[0m22:31:55.254078 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x104c3a660>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106cf3d90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106cf3ed0>]}


============================== 22:31:55.256701 | 6aa7387b-6b2d-4a5e-a881-69e30e58d8f9 ==============================
[0m22:31:55.256701 [info ] [MainThread]: Running with dbt=1.8.8
[0m22:31:55.256998 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': '/Users/maxhilsdorf/.dbt', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt docs serve', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m22:31:55.257279 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /Users/maxhilsdorf/Projekte/fortbildung/dbt_schibsted/data_pipeline_repo/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m22:31:55.263481 [debug] [MainThread]: Resource report: {"command_name": "serve", "command_wall_clock_time": 0.04080546, "process_user_time": 0.563763, "process_kernel_time": 0.081145, "process_mem_max_rss": "98566144", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m22:31:55.264043 [debug] [MainThread]: Command `dbt docs serve` failed at 22:31:55.263837 after 0.04 seconds
[0m22:31:55.264326 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106d0dba0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x106bbad50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1068fbce0>]}
[0m22:31:55.264510 [debug] [MainThread]: Flushing usage events
